phpunit.xml.dist

```diff
@@ -0,0 +1,14 @@
+<phpunit
+	bootstrap="tests/bootstrap.php"
+	backupGlobals="false"
+	colors="true"
+	convertErrorsToExceptions="true"
+	convertNoticesToExceptions="true"
+	convertWarningsToExceptions="true"
+	>
+	<testsuites>
+		<testsuite>
+			<directory prefix="test-" suffix=".php">./tests/</directory>
```

Comment:
Since we have both JS and PHP tests, I think we should call this directory something other than `tests`.  WP core uses `tests/phpunit/` and `tests/qunit/`, but our JS tests are inline with the files being tested.  Maybe we just call ours `phpunit/ instead?